com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbePlacement.cs

```diff
@@ -3,236 +3,443 @@
         {
-            foreach (ProbeReferenceVolume.Volume v in cullers)
+            using (new ProfilingScope(cmd, new ProfilingSampler("Rasterize Meshes 3D")))
             {
-                ProbeReferenceVolume.Volume lv = v;
+                var cellAABB = cellVolume.CalculateAABB();
 
-                foreach (ProbeReferenceVolume.Volume c in cullees)
+                using (new ProfilingScope(cmd, new ProfilingSampler("Clear")))
                 {
-                    if (result.Contains(c))
-                        continue;
+                    cmd.SetComputeTextureParam(subdivideSceneCS, s_ClearKernel, _Output, sceneSDF);
+                    DispatchCompute(cmd, s_ClearKernel, sceneSDF.width, sceneSDF.height, sceneSDF.volumeDepth);
+                }
+
+                cmd.SetRandomWriteTarget(4, sceneSDF);
 
-                    ProbeReferenceVolume.Volume lc = c;
+                var mat = new Material(Shader.Find("Hidden/ProbeVolume/VoxelizeScene"));
+                mat.SetVector(_OutputSize, new Vector3(sceneSDF.width, sceneSDF.height, sceneSDF.volumeDepth));
+                mat.SetVector(_VolumeWorldOffset, cellAABB.center - cellAABB.extents);
+                mat.SetVector(_VolumeSize, cellAABB.size);
 
-                    if (ProbeVolumePositioning.OBBIntersect(ref lv, ref lc))
-                        result.Add(c);
+                var topMatrix = GetCameraMatrixForAngle(Quaternion.Euler(90, 0, 0));
+                var rightMatrix = GetCameraMatrixForAngle(Quaternion.Euler(0, 90, 0));
+                var forwardMatrix = GetCameraMatrixForAngle(Quaternion.Euler(0, 0, 90));
+
+                Matrix4x4 GetCameraMatrixForAngle(Quaternion rotation)
+                {
+                    cellVolume.CalculateCenterAndSize(out var center, out var size);
+                    Vector3 cameraSize = new Vector3(sceneSDF.width, sceneSDF.height, sceneSDF.volumeDepth) / 2.0f;
+                    cameraSize = size / 2;
+                    var worldToCamera = Matrix4x4.TRS(Vector3.zero, rotation, Vector3.one);
+                    var projection = Matrix4x4.Ortho(-cameraSize.x, cameraSize.x, -cameraSize.y, cameraSize.y, 0, cameraSize.z * 2);
+                    return Matrix4x4.Rotate(Quaternion.Euler((Time.realtimeSinceStartup * 10f) % 360, 0, 0));
                 }
+
+                // We need to bind at least something for rendering
+                cmd.SetRenderTarget(dummyRenderTarget);
+                cmd.SetViewport(new Rect(0, 0, dummyRenderTarget.width, dummyRenderTarget.height));
+                var props = new MaterialPropertyBlock();
+                foreach (var kp in renderers)
+                {
+                    // Only mesh renderers are supported for the voxelization.
```

Comment:
This might be a problem for terrain.

Reply:
Will do something for terrain in the next PR